src/main/java/eu/phiwa/dragontravel/core/DragonManager.java

```diff
@@ -238,12 +245,17 @@ else if (dismountAtCurrentLocation || !DragonTravel.getInstance().getConfig().ge
             player.teleport(startLoc);
             player.sendMessage(DragonTravel.getInstance().getMessagesHandler().getMessage("Messages.General.Successful.DismountedToStart"));
             DragonPlayerDismountEvent event = new DragonPlayerDismountEvent(player, dragon, startLoc);
+            this.dismountedPlayer.add(player);
             Bukkit.getPluginManager().callEvent(event);
+            Bukkit.getScheduler().runTaskLater(DragonTravel.getInstance(), DragonManager.getDragonManager().removeFromDismountedList(player), 1L);
+
         }
 
         CheatProtectionHandler.unexemptPlayerFromCheatChecks(player);
     }
-    
+
+
+
```

Comment:
Could you remove two of those empty lines?

Reply:
done